fabric-item-api-v1/src/main/java/net/fabricmc/fabric/impl/client/ItemUpdateAnimationHandlerAccessor.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.client;
+
+import net.fabricmc.fabric.api.client.item.v1.UpdateAnimationHandler;
+
+public interface ItemUpdateAnimationHandlerAccessor {
+	UpdateAnimationHandler get();
+	void set(UpdateAnimationHandler handler);
```

Comment:
should be prepended with `fabric_` and named to specify they get and set the handlers